packages/ui/src/theme/css/component/inAppMessaging/backdrop.scss

```diff
@@ -0,0 +1,11 @@
+.amplify-inappmessaging-backdrop {
+  align-items: center;
+  background-color: var(--amplify-colors-overlay-50);
```

Comment:
Can we create a backdrop token for background-color and use that here? (which would reference --amplify-colors-overlay-50)